src/EventStore.Core/Services/Transport/Http/Controllers/PersistentSubscriptionController.cs

```diff
@@ -677,6 +678,27 @@ public class PersistentSubscriptionController : CommunicationController {
 				groupname, http.User);
 			Publish(cmd);
 		}
+		
+		private void RestartPersistentSubscriptions(HttpEntityManager http, UriTemplateMatch match) {
+			if (_httpForwarder.ForwardRequest(http))
+				return;
+
+			var envelope = new SendToHttpEnvelope(_networkSendQueue, http,
+				(e, message) => e.ResponseCodec.To("Restarting"),
+				(e, message) => {
+					switch (message) {
+						case SubscriptionMessage.PersistentSubscriptionsRestarting _:
+							return Configure.Ok(e.ResponseCodec.ContentType);
+						case SubscriptionMessage.InvalidPersistentSubscriptionsRestart fail:
+							return Configure.BadRequest
+								($"Persistent Subscriptions cannot be restarted as it is in the wrong state: {fail}");
```

Comment:
```suggestion
								($"Persistent Subscriptions cannot be restarted as it is in the wrong state.");
```
There isn't any relevant information on the message to show in this error message.